src/scripts/main.js

```diff
@@ -1,3 +1,182 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const thead = document.querySelector('thead');
+
+const pushNotification = (posTop, posRight, title, description, type) => {
+  const body = document.querySelector('body');
+
+  const message = document.createElement('message');
+
+  message.insertAdjacentHTML('afterbegin', `
+  <h2 class = title>${title}</h2>
+  <p>${description}</p>
+  `);
+
+  message.className = `notification ${type}`;
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  body.append(message);
+
+  setTimeout(() => {
+    message.remove();
+  }, 2000);
+}; // notifications
+
+tbody.addEventListener('click', (eve) => {
+  for (const tr of [...tbody.children]) {
+    if (tr.className === 'active') {
```

Comment:
theoretically, it can have other classes as well

```suggestion
    if (tr.classList.includes('active')) {
```